src/_guides/get-started.md

```diff
@@ -20,63 +21,26 @@ src="{{site.custom.dartpad.embed-dart-prefix}}?horizontalRatio=70&verticalRatio=
 Note that DartPad supports only a few core libraries.
 If you want to use other libraries,
 such as dart:io or libraries from packages,
-you'll need to [install](/install) an SDK.
+you'll need to download platform-specific tools.
 
 
-<div class="get-started-table__text">
-  <h2>Ready for something more?</h2>
-  <p>
-    This website (<a href="/">www.dartlang.org</a>) contains
-    information about the fundamental Dart technologies:
-  </p>
-  <ul>
-    <li>the <a href="/guides/language">language</a></li>
-    <li><a href="/guides/libraries">core libraries</a></li>
-  </ul>
-  <p>
-    But where do you go when you have a use case in mind?
-  </p>
-</div>
+## What next?
 
-| | | **Use case** | **Get started** |
-| <i class="fa fa-code" aria-hidden="true"></i> | **Web** | Create an app that runs in any modern browser | <a href="{{site.webdev}}/guides/get-started" class="btn btn-primary no-automatic-external">Dart webdev</a> |
-| <i class="fa fa-android" aria-hidden="true"></i> <i class="fa fa-apple" aria-hidden="true"></i> | **Mobile** | Create an app from a single codebase that runs on both iOS and Android | <a href="https://flutter.io/getting-started/" class="btn btn-primary no-automatic-external">Flutter</a> |
-| <i class="fa fa-terminal" aria-hidden="true"></i> | **Server** | Create a command-line app or server | <a href="/tutorials/dart-vm/get-started" class="btn btn-primary">Dart VM</a> |
-{:.get-started-table}
-
-{% comment %}
-<sup>*</sup>If you have a favorite IDE, there is probably a Dart plugin for that.
-See the link for tips on how to configure the recommended IDE.
-For general IDE advice, see [Tools](/tools).
-{% endcomment %}
-
-<div style='clear:right'></div>
-
-## Helpful flowchart
-
-{% comment %}
-The original Lucid Chart is here:
-https://www.lucidchart.com/documents/edit/0f170001-d5f4-4b17-8cdc-72d5575f2e78?shared=true&#
-Created by filiph, the Lucid app is available to Googlers in mystuff.
-{% endcomment %}
+To learn about the Dart [language](/guides/language) and
+[core libraries](/guides/libraries),
+stay on this site.
```

Comment:
[nit]: `"Learn about the Dart [language] and [core libraries] on this site. Once you have a use case ...."`